docs/campaign.rst

```diff
@@ -290,6 +290,27 @@ value (e.g. `--exclusvie`) is to be used in `srun`, the key should be assigned t
           exclusive: true
         type: osu
 
+spack (optional)
+~~~~~~~~~~~~~~~~
+Dictionary to specify spack related configuration. Supported attributes are:
+
+* **specs**: list of spack specs to install before executing benchmarks.
+  `bin` directory of install directories are be prepended to `PATH`.
+
+For instance:
+
+.. code-block:: yaml
+  :emphasize-lines: 5-7
+
+  benchmarks:
+      '*':
+          test01:
+              type: stream
+              spack:
+                  specs:
```

Comment:
As mentioned also in the other repo. do we really need this additional level. It seems redundant, or do you think in the future we might have other spack functions that would go in there?

Reply:
Thank you for the feedback. I agree this extra level is a pain, but as I am not expert at spack, I prefered to leave space for future changes.